Sources/SourceControl/RepositoryManager.swift

```diff
@@ -13,14 +13,15 @@ import class Foundation.OperationQueue
 
 import TSCBasic
 import TSCUtility
+import Basics
 
 /// Delegate to notify clients about actions being performed by RepositoryManager.
 public protocol RepositoryManagerDelegate: class {
     /// Called when a repository is about to be fetched.
-    func fetchingWillBegin(handle: RepositoryManager.RepositoryHandle)
```

Comment:
@abertelrud this need a default implementation too, right?

Reply:
Technically, all of the methods in the delegate protocol are optional, so they should have default implementations.  But since `fetchWillBegin(handle:)` is an existing method that any delegate must already be implementing, I don't think it's needed for this PR.

Before tagging a semver 1.0 of libSwiftPM we should revisit and standardize all of these, but since the method preexisted this PR I don't think anything else is needed here.